index.html

```diff
@@ -1,34 +1,40 @@
 <!DOCTYPE html>
-<html>
+<html lang="de">
 <head>
-<title>DU4L.org</title>
-<link rel="stylesheet" href="style.css">
-<link rel="shortcut icon" type="image/x-icon" href="icon_ctf.png">
-<link rel="icon" type="image/png" href="icon32x32_ctf.png" sizes="32x32">
-<link rel="icon" type="image/png" href="icon64x64_ctf.png" sizes="96x96">
-<link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png">
+	<meta charset="utf-8">
+	<title>DU4L.org</title>
+	<link rel="stylesheet" href="style.css">
+	<link rel="shortcut icon" type="image/x-icon" href="icon_ctf.png">
+	<link rel="icon" type="image/png" href="icon32x32_ctf.png" sizes="32x32">
+	<link rel="icon" type="image/png" href="icon64x64_ctf.png" sizes="96x96">
+	<link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png">
 </head>
 <body>
-    <main class="bsod container">
-        <h1 class="neg title"><span class="bg">DU4L - <span id='text'><noscript>@DHBW Mannheim</noscript></span></span></h1>
-        <p>W3llcome to the space of our CTF Team called DU4L. We are a team of students from the DHBW Mannheim, Germany.</p>
-        <p>* Contact us for questions.<br/>
-        * Look into our GitHub.</p>
-        <img class="center" src="logo-resize.png" alt="Dual Logo">
-        <p>Contact</p>
-        * Mail: contact [at] du4l [dot] org
-
-        
-        <nav class="nav">
-          <a href="https://github.com/DU4L" class="link">GitHub</a>&nbsp;|&nbsp;<a href="https://ctftime.org/team/170921" class="link">CTFtime</a>&nbsp;|&nbsp;<a href="impressum.html" class="link">Impressum</a>
-        </nav>
-        <br>
-        
-        
-      </main>
-      
-      
-
+	<div class="bsod container">
+		<h1 class="neg title">
+			<span class="bg">DU4L - 
+				<span id='typewriterText'>@DHBW Mannheim</span>
```

Comment:
hier war mal ein `noscript` tag. Jetzt wo der nicht mehr da ist flickert kurz der `@DHBW Mannheim` Text bevor das Javascript ausgeführt wird. Gibt's nen Grund den zu entfernen?

Reply:
Also ich habe ihn eigentlich als unnötig angesehen, weil auch alles ohne ihn funktioniert.
Ich sehe das Flickern aber irgendwie auch nicht.
Aber falls es dich stört, kannst du den da wieder dazwischen schreiben, funktioniert auch alles mit dem Tag.
`elementId.innerHTML = ''` _(in main.js typeWriter())_ wird den Tag aber auch direkt entfernen, wenn der User JavaScript im Browser aktiviert hat. Wenn nicht bleibt der Tag halt da.

Reply:
> Ich sehe das Flickern aber irgendwie auch nicht. 

Ja, passiert nur bei sehr langsamen Geräten, da das Javascript meistens schneller als die 16ms (60Hz) ist. Ich machs wieder dazu, schaden tuts nicht.